components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -638,8 +638,11 @@ def getJSToNativeConversionInfo(type, descriptorProvider, failureCode=None,
     # Note that we can't just set failureCode to exceptionCode, because setting
     # failureCode will prevent pending exceptions from being set in cases when
     # they really should be!
+    unhandledExceptionCode = exceptionCode
     if exceptionCode is None:
-        exceptionCode = "return false;"
+        exceptionCode = 'return false;'
+        unhandledExceptionCode = 'debug!("Unhandled exception error"); return false;'
```

Comment:
(Abort in the sense of stop the execution of the script, not in the sense of `SIGABRT`).